apps/web-giddh/src/app/reports/components/sales-register-component/sales.register.component.html

```diff
@@ -1,119 +1,120 @@
+                                        </select>
+                                    </div>
+                                </li>
 
-                <div class="d-inline-block">
-                    <div class="btn-group d-block" dropdown>
-                        <a href="javascript:void(0)" id="button-basic" dropdownToggle type="button"
-                            class="dropdown-icon d-inline-block" aria-controls="dropdown-basic">
-                            <i class="icon-path"></i>
-                        </a>
-                        <ul id="dropdown-basic" *dropdownMenu class="dropdown-menu" role="menu"
-                            aria-labelledby="button-basic">
-                            <li>
-                                <a href="javascript:void(0)" class="dropdown-item">
-                                    <input type="checkbox">
-                                    Voucher Type</a>
-                            </li>
-                            <li>
-                                <a href="javascript:void(0)" class="dropdown-item">
-                                    <input type="checkbox">
-                                    Voucher No.</a>
-                            </li>
-                            <li>
-                                <a href="javascript:void(0)" class="dropdown-item">
-                                    <input type="checkbox">
-                                    Product/Service</a>
-                            </li>
-                            <li>
-                                <a href="javascript:void(0)" class="dropdown-item">
-                                    <input type="checkbox">
-                                    Qty-Rate</a>
-                            </li>
-                            <li>
-                                <a href="javascript:void(0)" class="dropdown-item">
-                                    <input type="checkbox">
-                                    Value</a>
-                            </li>
+                                <li>
+                                    <div class="select-style width-200">
+                                        <select name="">
+                                            <option value="">Monthly</option>
+                                            <option value="">Weekly</option>
+                                            <option value="">Custom Range</option>
+                                        </select>
+                                    </div>
+                                </li>
+                                <li>
+                                    <a href="javascript:void(0)" class="exand-icon d-inline-block middle"><i class="icon-expand"></i></a>
+                                </li>
+                            </ul>
+    </div>
+    <div class="container-fluid font-14">
+        <div class="row mt-2">
+            <div class="col-xs-6">
```

Comment:
add col-6 or col-sm-6 in place of col-xs-6 @shalinee-walkover

Reply:
Done